HISTORY.rst

```diff
@@ -86,6 +86,8 @@ Unreleased Changes
     * Removed the GDAL cache size limit on this model, which means that, by
       default, the model will use up to 5% of installed memory.
       https://github.com/natcap/invest/issues/1320
+    * Fixed a bug where nodata pixels in quick flow were being set to 0
```

Comment:
Hey @emlys, I'm thinking of future us and wondering if we could overly expand on some context here. Mostly, because the prior implementation appears to have been a design decision by someone.  Maybe something like:

>Allowing monthly quick flow nodata values to propagate instead of being set to 0. The design choice to set to 0 was not well documented and was causing confusion about why some outputs had nodata values where other did not. By allowing nodata to propagate instead we're erroring on the side of not assuming data where this is not which could lead to misinterpretations of the results.

Not the best wordsmithing!

Reply:
I expanded the note along these lines. I think the link to the issue also provides some context.